firefox_puppeteer/testcase.py

```diff
@@ -4,8 +4,8 @@
 
```

Comment:
Great. But I would really suggest to keep the subfolder and put this file into firefox_puppeteer/testcases/. Maybe we can keep using `base.py`?